assets_manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "ai_assets": [
+    {
+      "dart_enum_name": "smbertVocab",
+      "url_suffix": "smbert_v0000/vocab.txt",
+      "path": "data/smbert_v0000/vocab.txt"
+    },
+    {
+      "dart_enum_name": "smbertModel",
+      "url_suffix": "smbert_v0000/smbert.onnx",
+      "path": "data/smbert_v0000/smbert.onnx",
+      "chunk_size": "10MB"
```

Comment:
I think we could use 11 here, so we can have fragments that are more similar in size (the last chunk will be ~8mb instead of ~2)